package.json

```diff
@@ -13,7 +13,10 @@
   "scripts": {
     "start": "node build/server.js",
     "test": "cake --use-js tests",
-    "build": "cake build"
+    "build": "cake build",
+    "lint": "npm run lint1 || npm run lint2",
```

Comment:
mmh, if you use the || instead of the &&, wont it prevent using the command in scripts ? 

(like npm run build && npm run publish)